ui/components/component-library/avatar-badge/avatar-badge.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Box from '../../ui/box/box';
+
+export const badgePositions = ['top', 'bottom'];
+
+export const AvatarWithBadge = ({
+  children,
+  badgePosition,
+  badge,
+  ...props
+}) => {
+  return (
+    <Box className="avatar-badge" {...props}>
+      {/* Generally the AvatarAccount */}
+      {children}
+      <Box
+        className={
+          badgePosition === 'top'
+            ? 'avatar-badge-token-position-top'
+            : 'avatar-badge-token-position-bottom'
+        }
```

Comment:
Use BEM naming conventions here `avatar-with-token__token--position-top`

Reply:
Updated it to `avatar-with-badge__token--position-top` and `avatar-with-badge__token--position-bottom`